### PR TITLE
Updated showing of member's picture (if github is configured)

### LINKED ISSUE #108: Member's display pic updation (if github configured).
**/team/members :**

If user account has some github username, then github's dp should be shown instead of no-image

### PR DESCRIPTION
**Pull request by namangupta01**
Closes #108

### CODE REVIEW

**Comment by namangupta01** [comment]
This is how its look like now
screen shot 2018-01-19 at 3 09 47 am

**Comment by AnkushMalik** [comment]
hi, you need to do the same in members#edit view. :smiley_cat: :+1:

**Comment by namangupta01** [comment]
@AnkushMalik Yes! Thank you :)

File: app/views/team/members/edit.html.slim [hunk 1]
```diff
@@ -5,6 +5,8 @@
         = link_to team_member_path(@member), class: 'image' do
```
Comment (DavidKang) [inline app/views/team/members/edit.html.slim]: This code is very similar to https://github.com/openSUSE/agile-team-dashboard/pull/109/files#diff-15851dbd35c1f7d2e90d4f44628e0754L4, this should be moved into partial.

Comment (namangupta01) [reply app/views/team/members/edit.html.slim]: @DavidKang Yes, will update. 👍

**Comment by Ana06** [comment]
@namangupta01 what is the state of this?

**Comment by Ana06** [comment]
Closing this as @namangupta01 doesn't answer. I'll finish it in https://github.com/openSUSE/agile-team-dashboard/pull/123

**Pull request closed without merging**